tests/model_forms/tests.py

```diff
@@ -2574,6 +2574,17 @@ def test_model_clean_error_messages(self):
         self.assertFalse(form.is_valid())
         self.assertEqual(form.errors['__all__'], ['Global error message.'])
 
+        def test_error_messages_deep_copy(self):
+            class CustomCharField(CharField):
+                def __init__(self, **kwargs):
+                    kwargs['error_messages'] = {'invalid': 'Form custom error message.'}
+                    super().__init__(**kwargs)
+
+            field = CustomCharField()
+            field2 = copy.deepcopy(field)
+            self.assertIsInstance(field2, CustomCharField)
+            self.assertIsNot(field2.error_messages, field.error_messages)
```

Comment:
Please move this test to the `tests/forms_tests/tests/test_forms.py`.

Reply:
sorry, wrong push